src/server/utils/jsUtils.js

```diff
@@ -197,10 +197,20 @@ function retryAsync(promiseFunc, maxTries, delay, iteration) {
     });
 }
 
+function stripErrorColon(err) {
```

Comment:
If we need to later remove the 'error:' string, then probably we could not add it as part of the error message.